bigquery_etl/shredder/config.py

```diff
@@ -462,6 +441,12 @@ def fields(self) -> tuple[str, ...]:
     ("telemetry_stable.prio_v4", ID),
 }
 
+# names of tables in glean *_derived datasets to ignore
+GLEAN_DERIVED_IGNORE_TABLES = {
+    # to be added in https://mozilla-hub.atlassian.net/browse/DENG-2089
+    "clients_last_seen_joined_v1",
```

Comment:
@BenWu The `clients_last_seen_joined` tables are also meant to be shredded, that's the intention of DENG-2089. I updated the doc, excuses for the confusion, I thought it had been updated already.

Reply:
Oh I thought that ticket was meant to be completed at a later time, after this was completed.  That means there's no need for an ignore list right now.
Unified metrics is only `moz-fx-data-shared-prod.telemetry_derived.unified_metrics_v1` right?

Reply:
Yes, correct.